### PR TITLE
📣add🆕favorite icon 💻/📱

### PR DESCRIPTION
**Pull request by mariobodemann**
...addressing #5 ... and having fun with PR descriptions. 😂

### CODE REVIEW

**Comment by cketti** [comment]
In my opinion the logo doesn't work very well as favicon. Maybe we should go for a simpler image?
At the very least we should get rid of the blank border.

**Comment by mariobodemann** [comment]
I ✂ the borders. I'll think about 🖼🎨 some more ...

File: tools/generate_favicons.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Please exevute me from the img folder to regenerate the favicons.
```
Comment (cketti) [inline tools/generate_favicons.sh]: exevute -> execute

**Comment by cketti** [comment]
lgtm

**Pull request merged**